modules/display.c

```diff
@@ -414,7 +381,7 @@ static gboolean brightness_fade_timeout_cb(gpointer data)
 
 	if ((cached_brightness <= 0) && (target_brightness != 0)) {
 		backlight_ioctl(FB_BLANK_UNBLANK);
-		dpms_set_display_on(TRUE);
```

Comment:
I'd like to keep the `dpms_set_display_on` name, at least semantically, explaining that it sets the state on. We went over this at length on the initial PR for the DPMS code. ;)